code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -304,7 +305,7 @@
 	desc = "A rose is a rose, by any other name."
 	special = "Use this weapon to change it's damage type between red, white and pale."	//like a different rabbit knife. No black though
 	icon_state = "rosered"
-	force = 70 //Less damage, can swap damage type
+	force = 80 //Less damage, can swap damage type
```

Comment:
80 pale damage is bullshit and too much. TOO much, Kirie.

Reply:
Its not 80 pale, its 80 red, 70 white and 50 pale.
Still kinda meh overall because it doesnt do fuckloads of damage like Justitia and Da Capo